operations/helm/charts/mimir-distributed/templates/distributor/distributor-dep.yaml

```
@@ -47,6 +47,9 @@ spec:
             - "-target=distributor"
             - "-config.expand-env=true"
             - "-config.file=/etc/mimir/mimir.yaml"
+            # Force gRPC clients to reconnect periodically in order to have them re-resolve endpoints after a scale up event
```

It's not just a scale up. Similar issues happen during rollouts. I would just say it in a generic way, for example:

```suggestion
            # Force gRPC clients connecting to distributor to reconnect periodically in order to have them re-resolve endpoints and discover new replicas.
```